readme.md

```diff
@@ -29,10 +29,29 @@ gulp.task('build', () => {
 
 This plugin will build the cordova project for the iOS platform.
 
-### Re-adding the iOS platform
+## API
 
-The ```ios()``` method accepts one optional parameter. If the parameter passed in is ```true```, it will first
-remove the entire iOS platform and add it again.
+### ios(options)
+
+#### options
+
+Type: `object`
+
+A key-value pair object where the key is the name of the preference and the value the value. The keys and types are described below.
+
+##### options.reAdd 
+
+Type: `boolean`
+
```

Comment:
Remove this empty line and add 2 spaces after `boolean`. This will prevent the lines being rendered at the same line.